spec.html

```diff
@@ -24747,7 +24747,7 @@ <h1>Forbidden Extensions</h1>
         Neither mapped nor unmapped arguments objects may be created with an own property named *"caller"*.
       </li>
       <li>
-        The behaviour of the following methods must not be extended except as specified in ECMA-402: `Object.prototype.toLocaleString`, `Array.prototype.toLocaleString`, `Number.prototype.toLocaleString`, `Date.prototype.toLocaleDateString`, `Date.prototype.toLocaleString`, `Date.prototype.toLocaleTimeString`, `String.prototype.localeCompare`, %TypedArray%`.prototype.toLocaleString`.
+        The behaviour of built-in methods which are specified in ECMA-402, such as `toLocaleString`, must not be extended except as specified in ECMA-402.
```

Comment:
I'm not finding a better solution but I wish the current text could be more explicit in saying it refers to many `toLocaleString`. Currently it gives an impression there is only method with this name.

Perhaps: 

```suggestion
        The behaviour of built-in methods which are specified in ECMA-402, such as those named `toLocaleString`, must not be extended except as specified in ECMA-402.
```

Reply:
I like it!